web/src/components/Calendar.js

```diff
@@ -0,0 +1,307 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import TableCell from '@material-ui/core/TableCell';
+import {
+    darken,
+    fade,
+    lighten,
+} from '@material-ui/core/styles/colorManipulator';
+import Typography from '@material-ui/core/Typography';
+import { EditingState, ViewState } from '@devexpress/dx-react-scheduler';
+import classNames from 'clsx';
+import {
+    Scheduler,
+    MonthView,
+    Appointments,
+    Toolbar,
+    DateNavigator,
+    AppointmentTooltip,
+    AppointmentForm,
+    EditRecurrenceMenu,
+    Resources,
+    DragDropProvider,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import { withStyles } from '@material-ui/core/styles';
+
+const getBorder = theme =>
+    `1px solid ${
+        theme.palette.type === 'light'
+            ? lighten(fade(theme.palette.divider, 1), 0.88)
+            : darken(fade(theme.palette.divider, 1), 0.68)
+    }`;
+
+const DayScaleCell = props => (
+    <MonthView.DayScaleCell
+        {...props}
+        style={{ textAlign: 'center', fontWeight: 'bold' }}
+    />
+);
+
+const styles = theme => ({
+    cell: {
+        color: '#78909C',
+        position: 'relative',
+        userSelect: 'none',
+        verticalAlign: 'top',
+        padding: 0,
+        height: 100,
+        borderLeft: getBorder(theme),
+        '&:first-child': {
+            borderLeft: 'none',
+        },
+        '&:last-child': {
+            paddingRight: 0,
+        },
+        'tr:last-child &': {
+            borderBottom: 'none',
+        },
+        '&:hover': {
```

Comment:
hovering shouldnt make it white, it should be the lighter grey / black (#272727)